### PR TITLE
chore(yarn): upgrade to yarn v3

### PR DESCRIPTION
**Pull request by schultzp2020**

### CODE REVIEW

**Review by kim-tsao — APPROVED** [review]

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ jobs:
       - name: Install dependencies
         shell: bash
         run: |
-          yarn install --frozen-lockfile
+          yarn install
```
Comment (nickboldt) [inline .github/workflows/release.yaml:47]: we don't want to release with a frozen lock file? we want the random internet resolution to take effect on every release ?

**Review by nickboldt — APPROVED** [review]

File: .github/workflows/release.yaml (hunk 1 above)
Comment (schultzp2020) [reply .github/workflows/release.yaml:47]: `--frozen-lockfile` has been replace with `--immutable` which is automatically enabled when it detects a CI environment,

**Pull request merged**